docs/graphql/manual/troubleshooting/index.rst

```diff
@@ -12,118 +12,58 @@ Troubleshooting Hasura GraphQL engine errors
+In this case, check the errors of the respective remote service. For actions, check out this :ref:`debugging page <debugging_actions>`.
 
-Error: no foreign constraint exists on the given column
--------------------------------------------------------
+GitHub issues
+-------------
 
-The Hasura GraphQL engine validates all the relationships (created using foreign key/manually) before it starts serving.
-When it encounters a relationship defined from table ``A -> B`` it looks for a foreign key constraint in table ``A``
-and when it can't find it, it throws the above error.
+It's possible that someone with the same problem has created a GitHub issue on the `Hasura repo <https://github.com/hasura/graphql-engine/issues>`__.
+If you don't come across an issue with solution in your search, feel free to create a `new issue <https://github.com/hasura/graphql-engine/issues/new>`__.
 
-Solution
-^^^^^^^^
+Hasura blog
+-----------
 
-- Connect to the database and switch to ``hdb_catalog`` schema.
-- In the ``hdb_relationship`` table, find the entry for the above relationship and delete it.
-- Restart GraphQL engine to verify.
+The `Hasura blog <https://hasura.io/blog/>`__ contains a lot of useful resources including tutorials. 
+You can use the search functionality to find what you're looking for.
 
-Error: field already exists
----------------------------
+.. admonition:: YouTube
 
-When a relationship is created using the Hasura GraphQL engine, it creates a special field with the relationship name
-which is used while fetching nested objects using GraphQL.
+  If you prefer to watch tutorials in the form of videos, check out the `Hasura Youtube channel <https://www.youtube.com/channel/UCZo1ciR8pZvdD3Wxp9aSNhQ>`__.
 
-Let's say we have tables called ``article`` and ``author`` as follows:
+Postgres docs
+-------------
 
-.. thumbnail:: /img/graphql/manual/troubleshooting/author_article.jpg
-  :alt: article author schema 
+If you come across a Postgres error, it will be helpful to check the `Postgres documentation <https://www.postgresql.org/docs/current/index.html>`__.
 
-Using the console if you have created a relationship named ``author`` from the ``article`` table to
-the ``author`` table, the Hasura GraphQL engine will create a special field ``author`` in the ``article`` table in its
-internal state. This field will be available via the GraphQL interface.
+Discord
+-------
 
-When this table is described using ``psql``, the ``author`` field will not be available as part of the list of fields
-returned by the describe command as it is something added by Hasura GraphQL engine. Now if a new column is created
-with the same name, i.e. ``author``, via ``psql``, the Hasura GraphQL engine will throw the above error when restarted as it has two
-references to the ``author`` field for the ``article`` table.
+If you didn't find a solution in any of the abovementioned sections or if you prefer to troubleshoot with the community,
```

Comment:
Typo in: "above mentioned"

Reply:
It's actually a word: https://www.merriam-webster.com/dictionary/abovementioned :D but it's fine for me to change :)

Reply:
Wow. TIL. In that case sorry for changing it :)